aws/lambda/python/sns_converter.py

```
@@ -5,36 +5,47 @@
 sns_topic_arn = os.environ["SNS_TOPIC_ARN"]
 client = boto3.client('sns')
 
+
 def lambda_handler(event, context):
```

Ruff

Missing return type annotation for public function `lambda_handler`

(ANN201)